api/envoy/admin/v3/config_dump.proto

```diff
@@ -49,13 +49,17 @@ message UpdateFailureState {
       "envoy.admin.v2alpha.UpdateFailureState";
 
   // What the component configuration would have been if the update had succeeded.
+  // This field may not be populated by xDS clients due to storage overhead.
   google.protobuf.Any failed_configuration = 1;
 
   // Time of the latest failed update attempt.
   google.protobuf.Timestamp last_update_attempt = 2;
 
   // Details about the last failed update attempt.
   string details = 3;
+
+  // This is the version of the rejected resource.
+  string version_info = 4;
```

Comment:
@markdroth

Reply:
I don't think so.  The nonce is only relevant in the xDS stream itself, not for reporting purposes.

Reply:
I am thinking from control plane's perspective. When this is used in CSDS, it's more for debugging purpose. So might be useful in that sense?

Reply:
I don't think so.  A nonce is only valid within the context of an individual xDS stream.  When that stream breaks and the client establishes a new one, the previous nonce is no longer relevant.  Given that, I don't see how it could be useful for reporting purposes from either the client or the server -- it's a completely ephemeral value.

Reply:
Make sense.